downstream/modules/hub/proc-configuring-the-client-to-verify-signatures.adoc

```diff
@@ -3,61 +3,67 @@
 
 = Configuring the client to verify signatures
 
+In order to ensure a container image pulled from the remote registry is properly signed, you must first configure the image with the proper public key in a policy file. 
```

Comment:
Suggested edit, can be ignored:

"To ensure a container image..." Remove "In order".